tracer/src/Datadog.Trace/Debugger/Instrumentation/MethodMetadataInfo.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="MethodMetadataInfo.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace.Debugger.Instrumentation
+{
```

Comment:
In overall we should need to decide an style for the repo, in this same PR you are using both styles of namespace declaration.

Reply:
Great point! Fixed in c16bf9327aa9d064a2aef7bc4483891f2073c8f8 (so that we're consistent with the block-style namespace declaration syntax, until we decide otherwise).